sdk/test-utils/perf/test/perfPolicy.spec.ts

```diff
@@ -33,10 +33,20 @@ export class PerfPolicyTest extends PerfTest<PerfPolicyOptions> {
     }
   };
   async run(): Promise<void> {
-    const targetUrl = url.parse(this.parsedOptions.url.value!);
-    const differentUrl = url.parse(this.parsedOptions.url.value!);
+    const urlOption = this.parsedOptions.url.value;
+
+    if (!urlOption) {
+      throw new Error(`URL not specified`);
+    }
+
+    const targetUrl = url.parse(urlOption);
+    const differentUrl = url.parse(urlOption);
```

Comment:
wait, why do we need both of these? could we re-assign the same value?

Reply:
(I know this wasn’t introduced by your PR per se, but I wonder if we can clean it up)

Reply:
Huh, yeah that is a bit weird, I'll tidy it up.

Reply:
actually, i see why now -- since the next line mutates `differentUrl.host`, calling `parse` separately for each variable ensures they are different objects, meaning that the mutation doesn't update `targetUrl.host` as well. There's probably some other workaround but I think not worth the hassle for this PR, unless you have something in mind :)